apps/stats/models.py

```diff
@@ -163,8 +161,8 @@ class Contribution(amo.models.ModelBase):
     annoying = models.PositiveIntegerField(default=0,
                                            choices=amo.CONTRIB_CHOICES,)
     is_suggested = models.BooleanField(default=False)
-    suggested_amount = DecimalCharField(max_digits=254, decimal_places=2,
-                                        nullify_invalid=True, null=True)
+    suggested_amount = models.DecimalField(max_digits=9, decimal_places=2,
```

Comment:
Moving from 254 to 9 might cause us issues if there's bad data in the DB, do we have a way to make sure that'll be ok?